iOS/README.md

```diff
@@ -9,7 +9,20 @@
 
 Ping provides these iOS samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-### Integrate with PingOne Advanced Identity Cloud / PingAM:
+The Ping SDK for iOS enables you to integrate your SPA with our services.
```

Comment:
`The Ping SDK for iOS enables you to integrate your iOS application with Ping services.`